Sources/UtilityBeltUITesting/Enumerations/LaunchArgument.swift

```diff
@@ -0,0 +1,8 @@
+// Copyright © 2021 SpotHero, Inc. All rights reserved.
+
+import Foundation
+
+/// An enumeration of launch arguments used when launching an XCUIApplication.
+public enum LaunchArgument {
+    case custom(value: String)
```

Comment:
if there is only one case here that is used to pass in a string, what about just changing the functions to take a string input instead? Or another option is to keep the previous enum with the xc ui case (and other predefined args we need in the future), and then provide two apis, one that takes strings (for custom use and more of a convenience api) and one that takes LaunchArguments and passes along the raw value to the string version.  

```swift
static func containsLaunchArgument(_ argument: LaunchArgument) -> Bool {
        return self.containsLaunchArgument(arguement.rawValue)
    }
}

static func containsLaunchArgument(_ arg: String) -> Bool {
     return self.processInfo.arguments.contains(arg)
}

Reply:
So Luke and I discussed this a bit in Slack, so for some context:
- We discussed avoiding strict cases like `xcuiTest` because this is a shared repository and different applications may want to signify that UI tests are taking place in different ways (e.x. see additional points below, which could be used instead, if desired)
- There are basically 3 type of formats in which one would send launch arguments:
  - Argument:value, which could be represented as `case foo(arg: String, value: String)` and which could evaluate to the string `"--arg value"`. This format could be used to do such things as telling the app to use a specific environment file (e.x. "--env production").
  - User defaults. There is a reserved syntax of `"-key value"` which will automatically inject the launch argument into user defaults. So that could be represented as `case userDefaults(key: String, value: String)`
  - Everything else, which is represented as `case custom(value: String)` and gives the application freedom for any custom values, such as `"XCUITesting"` or if they wanted instead: `"-isUITesting true"` or `"--env xcuitest"`

Reply:
@lquigley Probably not a bad idea to just add in those additional cases now

Reply:
gotcha, that makes sense to me.  It just looked odd how it currently is, with one enum case that essentially wraps a string.  

@lquigley any thoughts to adding the other cases in this PR?  It might make more sense and potentially highlight some areas of improvement?

Reply:
@khaptonstall It feels to me that those are beyond the scope of this card. What do you think about a new card and TODO?

Reply:
@lquigley I'd think it'd make sense to add it in this PR so that the `LaunchArgument` enum is complete as it seems low complexity (let me know if you disagree!). Additionally, I think adding those other cases will helper readers better understand the `custom` case and its use cases.